tutorindigo/templates/indigo/lms/static/js/dark-theme.js

```diff
@@ -1,8 +1,38 @@
 $(document).ready(function() {
     'use strict';
 
-    const enableDarkTheme = {% if INDIGO_ENABLE_DARK_THEME %}true{% else %}false{% endif %};
-    if (enableDarkTheme){
-        $('body').addClass("indigo-dark-theme");
+    let themeCookie = 'indigo-toggle-dark';
```

Comment:
I'm getting a build error during `tutor images build openedx`:

```
58.32 Traceback (most recent call last):                                                                                                                                                
58.32   File "/openedx/edx-platform/./manage.py", line 106, in <module>                                                                                                                 
58.33     execute_from_command_line([sys.argv[0]] + django_args)                                                                                                                        
58.33   File "/openedx/venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line                                                    
58.33     utility.execute()                                                                                                                                                             
58.33   File "/openedx/venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute                                                                      
58.33     self.fetch_command(subcommand).run_from_argv(self.argv)                                                                                                                       
58.33   File "/openedx/venv/lib/python3.11/site-packages/django/core/management/base.py", line 412, in run_from_argv                                                                    
58.33     self.execute(*args, **cmd_options)                                                                                                                                            
58.33   File "/openedx/venv/lib/python3.11/site-packages/django/core/management/base.py", line 458, in execute                                                                          
58.33     output = self.handle(*args, **options)                                                                                                                                        
58.33              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                        
58.33   File "/openedx/venv/lib/python3.11/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 209, in handle                                          
58.33     collected = self.collect()                                                                                                                                                    
58.33                 ^^^^^^^^^^^^^^                                                                                                                                                    
58.33   File "/openedx/venv/lib/python3.11/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 148, in collect                                         
58.33     for original_path, processed_path, processed in processor:                                                                                                                    
58.33   File "/openedx/venv/lib/python3.11/site-packages/require/storage.py", line 175, in post_process                                                                                 
58.33     for path in super_class.post_process(paths, dry_run, **options):                                                                                                              
58.33   File "/openedx/edx-platform/openedx/core/djangoapps/theming/storage.py", line 300, in post_process                                                                              
58.33     yield from super_class.post_process(paths.copy(), dry_run, **options)                                                                                                         
58.33   File "/openedx/venv/lib/python3.11/site-packages/pipeline/storage.py", line 39, in post_process                                                                                 
58.33     packager.pack_javascripts(package)                                                                                                                                            
58.33   File "/openedx/venv/lib/python3.11/site-packages/pipeline/packager.py", line 151, in pack_javascripts                                                                           
58.33     return self.pack(
58.33            ^^^^^^^^^^
58.33   File "/openedx/venv/lib/python3.11/site-packages/pipeline/packager.py", line 145, in pack
58.33     content = compress(paths, **kwargs)
58.33               ^^^^^^^^^^^^^^^^^^^^^^^^^
58.33   File "/openedx/venv/lib/python3.11/site-packages/pipeline/compressors/__init__.py", line 65, in compress_js
58.33     js = getattr(compressor(verbose=self.verbose), "compress_js")(js)
58.33          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
58.33   File "/openedx/venv/lib/python3.11/site-packages/pipeline/compressors/uglifyjs.py", line 10, in compress_js
58.33     return self.execute_command(command, js)
58.33            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
58.33   File "/openedx/venv/lib/python3.11/site-packages/pipeline/compressors/__init__.py", line 276, in execute_command
58.33     raise CompressorError(force_str(stderr))
58.33 pipeline.exceptions.CompressorError: Parse error at -:889,8
58.33 SyntaxError: Unexpected token: name (themeCookie)
58.33 Error
58.33     at new JS_Parse_Error (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:1545:18)
58.33     at js_error (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:1553:11)
58.33     at croak (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2089:9)
58.33     at token_error (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2097:9)
58.33     at unexpected (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2103:9)
58.33     at semicolon (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2123:56)
58.33     at simple_statement (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2314:73)
58.33     at eval (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2183:19)
58.33     at eval (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2136:24)
58.33     at block_ (eval at <anonymous> (/openedx/node_modules/uglify-js/tools/node.js:28:1), <anonymous>:2429:20)
58.33 
```

This issue is fixed by replacing "let" by "const". But please test your changes once again, now that we have made many changes to the implementation.

Reply:
Thanks for the fix. Updated.